python/semantic_kernel/connectors/memory/faiss.py

```diff
@@ -221,13 +219,9 @@ async def _inner_search_vectorized(
         )
 
 
-class FaissStore(VectorStore):
+class FaissStore(InMemoryVectorStore):
```

Comment:
Wouldn't now be a good time to collapse the memory-related imports to `from semantic_kernel.memory import FaissCollection`, etc...?